icecreamshop/src/main/java/com/summerschool/icecreamshop/dto/BasketProductDTO.java

```diff
@@ -12,6 +14,12 @@
     @NotNull
     private Double price;
 
+    @ElementCollection
+    private List<String> photoUrls;
+
+    @NotNull
+    private String title;
+
```

Comment:
Asta inseamna ca mereu cand voi adauga un produs in cos, va trebui sa trimit si numele produsului si lista de imagini? Ce se intampla cand vreau sa modific titlul sau imaginea unui produs care e adaugat in mai multe cosuri? Stocarea aceluiasi lucru in locuri diverse va crea (la un moment dat) incosistenta.